frontends/web/src/routes/market/swap/components/swap-service-selector.module.css

```
@@ -0,0 +1,105 @@
+}
+
+.select :global(.react-select__menu) {
+    background-color: var(--background-secondary);
+    z-index: 99;
+}
+
+.select :global(.react-select__menu) .swapServiceOption {
+    padding: 0 var(--space-half);
+}
+
+/*Hovered option color*/
```

Stylelint (17.7.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)